go.mod

```diff
@@ -1,6 +1,8 @@
 module go.k6.io/k6
 
-go 1.20
```

Comment:
I can separate this on another PR, but apparently it seems to be required now?
I guess it's related with https://go.dev/doc/toolchain